kubernetes.go

```diff
@@ -48,6 +48,10 @@ type KubernetesService interface {
 	GetOptions(context.Context) (*KubernetesOptions, *Response, error)
 	AddRegistry(ctx context.Context, req *KubernetesClusterRegistryRequest) (*Response, error)
 	RemoveRegistry(ctx context.Context, req *KubernetesClusterRegistryRequest) (*Response, error)
+
+	//Clusterlint support
```

Comment:
Nit: doesn't look like CI runs `golint` here, but if it did this comment would get a warning since it doesn't start with `// RunClusterlint ...`. I'd probably omit this comment entirely - we haven't delineated any other methods above in this way.